Rakefile

```diff
@@ -88,7 +88,7 @@ TEST_METADATA = {
     'graphql-2.1'  => '❌ 2.1 / ❌ 2.2 / ❌ 2.3 / ❌ 2.4 / ❌ 2.5 / ❌ 2.6 / ❌ 2.7 / ❌ 3.0 / ✅ 3.1 / ❌ 3.2 / ❌ 3.3 / ❌ jruby',
     'graphql-2.0'  => '❌ 2.1 / ❌ 2.2 / ❌ 2.3 / ❌ 2.4 / ❌ 2.5 / ❌ 2.6 / ❌ 2.7 / ❌ 3.0 / ✅ 3.1 / ❌ 3.2 / ❌ 3.3 / ❌ jruby',
     'graphql-1.13' => '❌ 2.1 / ❌ 2.2 / ❌ 2.3 / ❌ 2.4 / ❌ 2.5 / ❌ 2.6 / ❌ 2.7 / ❌ 3.0 / ✅ 3.1 / ❌ 3.2 / ❌ 3.3 / ❌ jruby',
-    'graphql-1.12' => '❌ 2.1 / ❌ 2.2 / ❌ 2.3 / ❌ 2.4 / ❌ 2.5 / ❌ 2.6 / ❌ 2.7 / ❌ 3.0 / ✅ 3.1 / ❌ 3.2 / ❌ 3.3 / ❌ jruby',
```

Comment:
This is surprising -- why can't we test graphql on other Ruby versions? :thinking: :eyes:

Reply:
Our 2.0 release contains breaking changes that makes existing graphql-ruby code not functional

Reply:
I was using 3.1 as an example and will update the matrix with other Ruby versions once the changes are confirmed. 

Also, the dependency should not be installing from my fork but the official gem release. 😄

Reply:
Got it -- thanks for clarifying :bow: